source/Octopus.Tentacle.Kubernetes.Tests.Integration/Setup/Tooling/KindDownloader.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using OperatingSystem = Octopus.Tentacle.Kubernetes.Tests.Integration.Setup.Tooling.OperatingSystem;
+
+namespace Octopus.Tentacle.Kubernetes.Tests.Integration.Setup.Tooling
+{
+    public class KindDownloader : ToolDownloader
+    {
+        const string LatestKindVersion = "v0.22.0";
```

Comment:
Does this determine the version of the cluster? Do you think we would need to test against the latest N minor versions?

Reply:
hrm, it doesn't. We should probably specify the version of the cluster we are testing against.

In terms of version testing, let's have an async chat about it in Slack

Reply:
I've added config to the `kind-config.yaml` to set each kind node to be version 1.29.2